buyer_data/utils/loan_math.py

```
@@ -0,0 +1,51 @@
+from math import pow, log
+
+def get_query_params(request, key, type_cast=float, required=True):
+    value = request.query_params.get(key)
+    if required and value is None: raise ValueError(f'Missing required query parameter: {key}')
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)